lib/dynamoid/persistence.rb

```diff
@@ -497,6 +498,8 @@ def save(options = {})
     def update_attributes(attributes)
       attributes.each { |attribute, value| write_attribute(attribute, value) }
       save
+    rescue Dynamoid::Errors::UnknownAttribute
+      false
```

Comment:
I think it makes sense to follow Rails convention and raise `UnknownAttribute` exception anyway even in methods without bang.

Reply:
Sounds good to me, I'll make the change